backend/src/mastodon/notification.ts

```diff
@@ -212,7 +212,7 @@ export async function getNotifications(db: D1Database, actor: Actor, domain: str
 		const notifFromAccount = await loadExternalMastodonAccount(acct, notifFromActor)
 
 		const notif: Notification = {
-			id: result.notif_id.toString(),
+			id: (result.notif_id as string).toString(),
```

Comment:
If we are casting to string, then why call `toString()` on it :-P ?

Reply:
Ideally we should provide a type to the `stmt.all()` function that defines the types expected from the query.
e.g. something like (after also defining `ObjectsRow`)

```
interface NotificationsQueryResult extends ObjectsRow {
  type,
  actor_id,
  notif_from_actor_id,
  notif_cdate,
  notif_id
}
```

Reply:
I just quickly temporarily added an inline type to make eslint happy, I'll try to see if a better type and be defined and shared 👍

Reply:
I've introduced the `NotificationsQueryResult` and `ObjectsRow` interfaces, please have a look 🙂